debug_toolbar/templates/debug_toolbar/redirect.html

```diff
@@ -4,8 +4,12 @@
   <head>
     <title>Django Debug Toolbar Redirects Panel: {{ status_line }}</title>
     <script{% if toolbar.csp_nonce %} nonce="{{ toolbar.csp_nonce }}"{% endif %} type="module" src="{% static 'debug_toolbar/js/redirect.js' %}" async></script>
+    <link{% if toolbar.csp_nonce %} nonce="{{ toolbar.csp_nonce }}"{% endif %} rel="stylesheet" href="{% static 'debug_toolbar/css/toolbar.css' %}">
   </head>
-  <body>
+  <body id="djDebug" {% if toolbar.store_id %}data-store-id="{{ toolbar.store_id }}"{% endif %}>
```

Comment:
The `id="djDebug"` seems to be causing downstream effects. I wonder if it'd be better to have the styles defined in the `<head>` here directly?